css/styleindex.css

```diff
@@ -0,0 +1,28 @@
+body{
+  background: url('imgs/bluecookies.jpg');
```

Comment:
This isn't working because this URL needs to be relative to the URL of the CSS file... since this file is inside the `css` directory at `css/styleindex.css`, the image file is up a level, at `../imgs/bluecookies.jpg`.